src/utils/level.ml

```diff
@@ -0,0 +1,48 @@
+type t = int
```

Comment:
Could we get away with `t` being abstract? Maybe if we expose a comparison function for levels?